fritzbox_wifi_devices.py

```diff
@@ -37,6 +40,8 @@ def get_connected_wifi_devices():
     if m:
         connected_devices = int(m.group(1))
         print 'wifi.value %d' % connected_devices
+    else:
+        print 'wifi.value 0'
```

Comment:
Do we really want to return '0' instead of nothing here?

Reply:
If the regex does not match, we do not have connected client (or at least, that's the case in my fritzbox). So, instead of returning nothing (which results in a `NaN`), I would say that it is cleaner to return 0. 

But, I do not use Wifi on my FritzBox, so it doesn't really matter for me.

Reply:
I personally prefer those "gaps": http://i.imgur.com/jEqUFI6.png
But 0 is fine for me too.

Reply:
I can make it configurable?

Reply:
I think that too much. I would say, we drop the zero. If you have WiFi enabled and there are no devices connected, it would show 0, wouldn't it?

Reply:
I doesn't for me, as it says "No wireless LAN devices" or something like that. Will remove it from this PR (it doesn't even really belong here)